design/one-pager-default-resource-class.md

```diff
@@ -3,21 +3,42 @@
 * Reviewers: Crossplane Maintainers
 * Status: Accepted
 
-The Crossplane ecosystem exposes the concepts of [Resource Classes and Resource Claims](https://crossplane.io/docs/v0.2/concepts.html). Classes serve to define the configuration for a certain underlying resource, which may be anything from a managed cloud provider service to a traditional server. Claims are requests to create an instance of a resource and currently reference a specific deployed Class in order to abstract the implementation details. This document serves to define the concept of a *Default Resource Class*, which allows for a more general abstraction of an underlying resource.
+The Crossplane ecosystem exposes the concepts of [Resource Classes and Resource
+Claims](https://crossplane.io/docs/v0.2/concepts.html). Classes serve to define
+the configuration for a certain underlying resource, which may be anything from
+a managed cloud provider service to a traditional server. Claims are requests to
+create an instance of a resource and currently reference a specific deployed
+Class in order to abstract the implementation details. This document serves to
+define the concept of a *Default Resource Class*, which allows for a more
```

Comment:
and support consuming multiple classes of service that can be consumed in a portable way across clouds